lib/rate-limiter.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Bluebird = require('bluebird');
+
+exports.limit = (request, client, defaultRate) => {
```

Comment:
thought the actual limiter should be separated - `index.js` seems like it should only have business logic to keep it easy to read